phantom-dsl/src/main/scala/com/websudos/phantom/util/ByteString.scala

```diff
@@ -29,13 +29,13 @@
  */
 package com.websudos.phantom.util
 
-import java.nio.{ ByteBuffer, ByteOrder }
-import java.lang.{ Iterable ⇒ JIterable }
+import java.lang.{Iterable => JIterable}
+import java.nio.{ByteBuffer, ByteOrder}
 
-import scala.collection.{mutable, IndexedSeqOptimized, immutable}
-import scala.collection.mutable.{ Builder, WrappedArray }
-import scala.collection.immutable.{ IndexedSeq, VectorBuilder }
 import scala.collection.generic.CanBuildFrom
+import scala.collection.immutable.{IndexedSeq, VectorBuilder}
+import scala.collection.mutable.WrappedArray
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/flavian/phantom_2/file/2220904841/issues/source?bid=2847375&fileBranchId=3173244#l37)